packages/kamino-sdk/src/Kamino.ts

```diff
@@ -439,6 +439,17 @@ export class Kamino {
     return getRebalanceMethodFromRebalanceFields(rebalanceFields);
   };
 
+  getReferencePriceTypeForStrategy = async (strategy: PublicKey | StrategyWithAddress): Promise<PriceReferenceType> => {
+    const strategyWithAddress = await this.getStrategyStateIfNotFetched(strategy);
```

Comment:
i think it would be cleaner to have a `numericalReferencePriceTypeToPriceTYpe` util, similar to `numberToDex` and use that everywhere

Reply:
agree